styles/chatgpt/catppuccin.user.css

```diff
@@ -526,39 +518,39 @@
     }
 
     .bg-gray-100 {
```

Comment:
is there any point of these and etc, are they not just restating the same CSS provided by the original source.

Reply:
Yep, that's the inefficiency of Tailwind for ya.

![Screenshot 2024-02-16 at 06 13 09 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/cbc07306-4bd9-429d-a015-946b0f578025)
![Screenshot 2024-02-16 at 06 13 02 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/f6e3b895-09de-46b7-ada4-c22400ef05c5)
![Screenshot 2024-02-16 at 06 12 32 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/aba66417-19a2-43e0-9dc0-7c920730eb78)

![Screenshot 2024-02-16 at 06 13 39 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/4e61ead6-2e91-450e-b8fe-81ad4242787f)

Reply:
I see, i'll mark this as resolved then.